4.next - Add docs for editor link features. (#6462)
cakephp/docs: en/development/debugging.rst

.. php:class:: Debugger

> To use the debugger, first ensure that ``Configure::read('debug')`` is set to

Reviewer: ```suggestion
To use the debugger, set ``Configure::read('debug')`` to
```

``true``. You can use ``filter_var(env('DEBUG', true),
FILTER_VALIDATE_BOOLEAN),`` in **config/app.php** file to ensure that ``debug``
is a boolean.

> The following configuration options in your **config/app.php** to change how

Reviewer: ```suggestion
The following configuration options can be set in **config/app.php** to change how
```

Get the type of a variable. Objects will return their class name

Editor Integration
==================

Exception and error pages can contain URLs that directly open in your editor or

> IDE. CakePHP ships with URL formats for several popular editors and you can add

Reviewer: ```suggestion
IDE. CakePHP ships with URL formats for several popular editors, and you can add
```
